### DISCUSSION TITLE
option to mv plot file and rename instead of copy and rename

### DISCUSSION THREAD

**Discussion by skunkworksguy** [madMAx43v3r/chia-plotter#715] [Ideas]
It would be really nice to have an option, or even just the standard functionality, to mv the final plot.tmp file to the final dir instead of copying, which is much more time consuming.

**Comment by Dartellum** [comment]
You can set final destination to your tempdir and then mv it yourself with either a cron job checking for a *.plot every X minutes or make a wrapper script to do it after creation. I could maybe make it an option on my wrapper script: https://github.com/Dartellum/chia-madmax-plot_loop.

**Comment by madMAx43v3r** [comment]
it'll try to do a rename first, only if that fails will do an actual copy
